odm2api/ODM2/models.py

```diff
@@ -1054,12 +772,15 @@ class Sites(Base):
     Longitude = Column('longitude', Float(53), nullable=False)
 
     SpatialReferenceObj = relationship(SpatialReferences)
-    SamplingFeatureObj = relationship(SamplingFeatures)
+    # SamplingFeatureObj = relationship(SamplingFeatures)
 
+    __mapper_args__ = {
+        'polymorphic_identity':'Site',
```

Comment:
Should it be `'polymorphic_identity':'sites',` (lowercase and plural)? I don't know myself, but that seems more consistent with what you're doing in `Results`